project/Dependencies.scala

```diff
@@ -190,6 +190,7 @@ object Dependencies {
     httpClient,
     googleApiClient,
     scalaUri,
+    workspaceManager,
```

Comment:
I am not so happy about this, but the current code requires it and I don't want to change it at the moment